model/cluster_states.go

```diff
@@ -11,6 +11,10 @@ const (
 	ClusterStateCreationRequested = "creation-requested"
 	// ClusterStateCreationInProgress is a cluster that is being actively created.
 	ClusterStateCreationInProgress = "creation-in-progress"
+	// ClusterStateWaitingForNode is a cluster that is waiting for nodes to be ready
+	ClusterStateWaitingForNode = "waiting-for-node"
```

Comment:
nit: I think this sounds slightly better, but we can also change it later
```suggestion
	// ClusterStateWaitingForNodes is a cluster that is waiting for nodes to be ready
	ClusterStateWaitingForNodes = "waiting-for-nodes"
```

Reply:
Right. Thanks